superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -61,102 +61,92 @@ const createFetchResourceMethod = (method: string) => (
   return [];
 };
 
-export const getRecentAcitivtyObjs = (
-  userId: string | number,
-  recent: string,
-  addDangerToast: (arg1: string, arg2: any) => any,
-) => {
-  const getParams = (filters?: Array<any>) => {
-    const params = {
-      order_column: 'changed_on_delta_humanized',
-      order_direction: 'desc',
-      page: 0,
-      page_size: 3,
-      filters,
-    };
-    if (!filters) delete params.filters;
-    return rison.encode(params);
+const getParams = (filters?: Array<any>) => {
```

Comment:
I wonder if we can type this more specifically.

Reply:
yep!